cookiecutter-frontend-app/cookiecutter.json

```diff
@@ -0,0 +1,8 @@
+{
+  "repo_owner": "edx",
+  "repo_name": "frontend-app-template",
+  "author_name": "{{cookiecutter.repo_owner}}",
```

Comment:
We can default this Github owner we already have, though it can still be customized.